app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -52,4 +54,15 @@ public function store(ReportbackRequest $request)
 
         return $this->item($reportback, $code);
     }
+
+    /**
+     * Update reportbackitem(s) that already exists.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function updateReportbackItems(Request $request)
+    {
+        $this->reportbackService->updateReportbackItems($request->all());
```

Comment:
@chloealee sorry! My instructions above are just for the what is done in your method in the Controller. The Service and Repository should just return the instance of the item or items  that was updated and maybe `null` if something went wrong.